addons/auth_oauth/views/auth_oauth_templates.xml

```diff
@@ -1,35 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+        <template id="auth_oauth.assets" inherit_id="web._assets_frontend_helpers" name="OAuth Providers Assets">
+            <xpath expr="//link[last()]" position="after">
+                <link rel="stylesheet" type="text/scss" href="/auth_oauth/static/src/scss/auth_oauth.scss"/>
+            </xpath>
+        </template>
+
         <template id="auth_oauth.providers" name="OAuth Providers">
-            <div t-foreach="providers" t-as="p">
-                <a t-att-href="p['auth_link']" class="btn btn-link" role="button">
-                    <i t-att-class="p['css_class']"/>
-                    <t t-esc="p['body']"/>
-                </a>
-            </div>
+            <t t-if="len(providers) &gt; 0">
+                <em class="d-block text-center text-muted small my-3">- or -</em>
+                <ul class="o_auth_oauth_providers list-group mt-1 mb-1 text-left">
+                    <a t-foreach="providers" t-as="p" class="list-group-item list-group-item-action py-2" t-att-href="p['auth_link']">
```

Comment:
`ul > a` structure is not valid HTML. Use a `<div/>` instead of `<ul/>` as BS suggests.